articles/azure/azs-how-add-disks.md

```diff
@@ -0,0 +1,247 @@
+---
+title: How to add disks on Azure Stack
+description: How to add disks on Azure Stack
+services: azure-stack
+author: David Woffendin
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2: Disk Management
+toc_sub3:
+toc_sub4:
+toc_title: How to add disks on Azure Stack
+toc_fullpath: Users/How To/Disk Management/azs-how-add-disks.md
+toc_mdlink: azs-how-add-disks.md
+---
+# Adding disks on Azure Stack
+
+## Overview
+
+In Azure Stack there are two types of disks, **Managed** and **Unmanaged** both these disks can be used on VM's as both OS and data disks.
+
+### Managed disks
+
+These disks simplify disk management for Azure IaaS VMs by managing the storage accounts associated with the VM disks. You only have to specify the size of disk you need, and Azure Stack creates and manages the disk for you.
+
+### Unmanaged disks
+
+These disks require that you create a storage account to store the disks. The disks you create are referred to as VM disks and are stored in containers in the storage account.
+
+## Using the portal to add new disks to VM's
+
+1. In the portal choose **Virtual machines**.
+
+2. Select the VM you wish to add disks too.
+
+3. Navigate to **Disks** > **+Add data disk**.
+
+    ![Azure Stack disk creation 1](images/azs-browser-disk-creation.png)
+
+4. Edit the options for your new Disk.
+
+    - **Name**: This is what the disk will be called.
+
+    - **Source type**: Leave this as **New(empty disk)** as we are creating a new disk.
+
+    - **Account type**: This is where you choose between SSD and HDD.
+
+    - **Size**: This is where you specify the size of the new disk.
+
+    - **Storage container**: This is where you specify the storage container for the disk. Find the storage account where your VM's OS disk is located and then create a new.
+
+    - **Container**. Select this container as the location for the new disk.
+
+    - **Storage blob name**: This is the blob where the disk will be stored.
+
+    ![Azure Stack disk creation 2](images/azs-browser-disk-creation2.png)
+
+5. Select Ok and the new disks will be added to the VM.
+
+## Using the portal to add disks to existing VM's
+
+1. Ensure the existing disk you want to add to your VM is on you local machine as a .vhd file. Upload this .vhd to a new Container in your VM's storage account.
+
+2. After the .vhd file is uploaded, you are ready to attach the VHD to a VM. In the menu on the left, select **Virtual machines**.
```

Comment:
After you've uploaded the .vhd file, you're ready...

Reply:
done